### PR TITLE
Some updates to installation texts

### PR DESCRIPTION
**Pull request by funkyfuture**
i did this mainly because the help text `http_port_helper` was rather confusing
as i didn't enter (or rather leave) the proper value there in a Docker setup,
resulting in a hard-to-troubleshoot installation. twice in the meantime.

otherwise i tried to improve the style and consistency.

### CODE REVIEW

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -83,21 +83,21 @@ general_title = General Application Settings
...
-domain = Domain
-domain_helper = This affects SSH clone URLs.
+domain = SSH Domain
+domain_helper = This domain name is used to construct SSH clone URLs.
 ssh_port = SSH Port
 ssh_port_helper = Port number which your SSH server is using. Leave it empty to disable.
-http_port = HTTP Port
-http_port_helper = Port number which application will listen on.
+http_port = Local HTTP Port
+http_port_helper = The port on which gitea serves the API in its execution context (e.g. a container).
```
Comment (lafriks) [inline options/locale/locale_en-US.ini]: It's not only API but all http requests, so instead of `API` it would probably be `web`

**Review by lafriks — CHANGES_REQUESTED** [review]

File: options/locale/locale_en-US.ini (hunk 1 above)
Comment (funkyfuture) [reply options/locale/locale_en-US.ini]: maybe there's a misunderstand, what is this value used for other than calls to the api by igitea itself? at least that was my conclusion when i realised i just had to leave it as `3000` when running from the provided Docker image while that port is mapped to another one externally.

Comment (lafriks) [reply options/locale/locale_en-US.ini]: It is the port gitea web will listen on for web/API/git http requests (localy) and http proxy in front needs to forward requests to this port if used.

Comment (funkyfuture) [reply options/locale/locale_en-US.ini]: a'ight, amended.

**Review by techknowlogick — APPROVED** [review]

**Review by appleboy — APPROVED** [review]

**Review by sapk — APPROVED** [review]

File: options/locale/locale_en-US.ini (hunk 1 above)
Comment (lafriks) [reply options/locale/locale_en-US.ini]: There should be space between web and interface

**Comment by funkyfuture** [comment]
`webinterface` -> `web interface` amended.

**Comment by lafriks** [comment]
@funkyfuture sorry for asking for one more change but first `its` in that sentence should be removed

**Comment by funkyfuture** [comment]
why should it? i'm not an english native speaker, but another have western germanic language as mother tongue. and to use a possesive pronoun is just appropriate in this context.

**Comment by lafriks** [comment]
But it is not good style to repeat words in same sentence and first one is not really that needed anyway, it just sounds weird :)

**Comment by bugreport0** [comment]
I've been working on the translation in #3614 … comments welcome.

**Comment by funkyfuture** [comment]
oh shit, sorry, i haven't seen this. i'm fine if you consider and incorprate what i proposed here. i give the install part a review then. deal?

**Comment by bugreport0** [comment]
@funkyfuture Yes, please review and see if I already solved your issue. Comments welcome. I just want to keep all English translation improvements in one place until we can merge them.

**Comment by funkyfuture** [comment]
closing this in favor of #3614.

**Pull request closed without merging**